__tests__/index.test.js

```diff
@@ -171,6 +132,55 @@ describe('GET /api/articles', () => {
     })
 })
 
+describe('GET /api/articles/:article_id', () => {
+    test('GETS an array of all articles with article id - status 200', () => {
+        return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.article).toMatchObject({
+                    author: "butter_bridge",
+                    title: "Living in the shadow of a great man",
+                    article_id: 1,
+                    body: "I find this existence challenging",
+                    topic: "mitch",
+                    created_at: "2020-07-09T20:11:00.000Z",
+                    votes: 100,
+                    article_img_url:
+                        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                })
+            })
+    })
+
+    test('sends an appropriate status and error message when given a valid but non-existent id - status 404', () => {
+        return request(app)
+            .get('/api/articles/999999')
+            .expect(404)
+            .then((response) => {
+                expect(response.body.msg).toBe("article_id '999999' not found");
+            });
+    });
+
+    test('sends an appropriate status and error message when given an invalid id', () => {
+        return request(app)
+            .get('/api/articles/not-an-article')
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toBe('Bad request');
+            });
+    });
+
+    test('article has correct comment count', () => {
+        return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.article.comment_count).toBe(11)
+            })
+    })
+})
```

Comment:
Great work, but there was no need to delete the tests from before and re add them. You can add the new test to the same endpoint and then add the logic to get it passing 😄